### PR TITLE
[REFACTOR] replace async_wrap with run_sync

### PR DESCRIPTION
**Pull request by dni**
cleans up code and needed for cln to be shutdown properly.

cherry picked from https://github.com/lnbits/lnbits/pull/1793/

### CODE REVIEW

**Review by motorina0 — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -74,7 +74,7 @@ openapi:
 	HOST=0.0.0.0 \
 	PORT=5003 \
 	poetry run lnbits &
-	sleep 7
+	sleep 15
```
Comment (callebtc) [inline Makefile:77]: Unrelated?

File: lnbits/server.py [hunk 2]
```diff
@@ -4,13 +4,10 @@
 
 import click
 import uvicorn
-import uvloop
 from uvicorn.supervisors import ChangeReload
 
 from lnbits.settings import set_cli_settings, settings
 
-uvloop.install()
-
```
Comment (callebtc) [inline lnbits/server.py:13]: Unrelated?

**Review by callebtc — CHANGES_REQUESTED** [review]
Possibly unrelated code.

File: lnbits/server.py (hunk 2 above)
Comment (jackstar12) [reply lnbits/server.py:13]: no, see: https://github.com/lnbits/lnbits/pull/1860/commits/42067945e5da4afbdc07271b1ad3784c5d56abc6

**Review by jackstar12 — APPROVED** [review]

**Review by talvasconcelos — APPROVED** [review]
tested.

**Pull request merged**